src/nemo_run/core/execution/dgxcloud.py

```diff
@@ -54,7 +51,8 @@ class DGXCloudExecutor(Executor):
     project_name: str
     container_image: str
     nodes: int = 1
-    gpus_per_node: int = 8
+    gpus_per_node: int = 0
+    nprocs_per_node: int = 0
```

Comment:
Should the default be 1 here? `nproc_per_node=0` seems a bit unintuitive.

Reply:
Good point. I think my original thinking was to catch if both `gpus_per_node` and `nprocs_per_node` were zero, but looking at this again, there isn't any benefit to that it appears, and agreed that defaulting to a single process per node makes more sense. Updated to 1 here.